File: trabalho 1/programas tarefa 1/java/micro/micro06.java
public class micro06{
    public static void main(String[] args){
        int numero;
        System.out.print("Digite um numero de 1 a 5: ");
        numero = Integer.parseInt(System.console().readLine());
        if(numero == 1){
            System.out.println("Um");
        }else if(numero == 2){
            System.out.println("Dois");
        }else if(numero == 3){
            System.out.println("Tres");
        }else if(numero == 4){
            System.out.println("Quatro");
        }else if(numero == 5){
            System.out.println("Cinco");
        }else{
            System.out.println("Numero invalido!!!");
        }
    }
}

File: trabalho 4/codigos/lua/micro/micro06.lua
function int main()
    int numero
    print("Digite um numero de 1 a 5: ")
    numero = io.read('*n')
    if numero == 1 then
        print("Um\n")
    end
    if numero == 2 then
        print("Dois\n")
    end
    if numero == 3 then
        print("Tres\n")
    end
    if numero == 4 then
        print("Quatro\n")
    end
    if numero == 5 then
        print("Cinco\n")
    else
        print("Numero invalido!!!")
    end
   return 1
end

main()
